app/javascript/src/components/editor/Empty.svelte

```diff
@@ -42,7 +41,6 @@
         <li>Variables that are only stated in a For Global (or Player) Variable Loop will not be properly compiled.</li>
         <li>Some actions or values may incorrectly indicate a wrong number of arguments.</li>
         <li>Rules without names might not be properly imported.</li>
-        <li>You can't rename projects.</li>
```

Comment:
You ever just add a feature and then forget to tell people you added the feature? Couldn't be me.

Reply:
:D